portal-kernel/test/unit/com/liferay/portal/kernel/workflow/DefaultWorkflowTransitionTest.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.kernel.workflow;
+
+import com.liferay.portal.kernel.language.Language;
+import com.liferay.portal.kernel.language.LanguageUtil;
+import com.liferay.portal.kernel.util.HashMapBuilder;
+import com.liferay.portal.kernel.util.LocaleUtil;
+
+import java.util.HashMap;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import org.mockito.Mockito;
+
+/**
+ * @author Feliphe Marinho
+ * @author Paulo Albuquerque
+ */
+public class DefaultWorkflowTransitionTest {
+
+	@Test
+	public void testGetLabel() {
+		LanguageUtil languageUtil = new LanguageUtil();
+
+		Language language = Mockito.mock(Language.class);
+
+		Mockito.when(
+			language.get(LocaleUtil.BRAZIL, "approve", null)
+		).thenReturn(
+			"Aprovar"
+		);
+
+		Mockito.when(
+			language.get(LocaleUtil.getSiteDefault(), "proceed")
+		).thenReturn(
+			"Proceed"
+		);
+
+		languageUtil.setLanguage(language);
+
+		DefaultWorkflowTransition defaultWorkflowTransition =
+			new DefaultWorkflowTransition();
+
+		defaultWorkflowTransition.setLabelMap(new HashMap<>());
+		defaultWorkflowTransition.setName(null);
+
+		Assert.assertEquals(
+			"Proceed",
+			defaultWorkflowTransition.getLabel(LocaleUtil.getSiteDefault()));
+
+		defaultWorkflowTransition.setLabelMap(new HashMap<>());
+		defaultWorkflowTransition.setName("approve");
+
+		Assert.assertEquals(
+			"Aprovar", defaultWorkflowTransition.getLabel(LocaleUtil.BRAZIL));
+
+		defaultWorkflowTransition.setLabelMap(new HashMap<>());
+		defaultWorkflowTransition.setName("businessRuleReview");
+
+		Assert.assertEquals(
+			"businessRuleReview",
+			defaultWorkflowTransition.getLabel(LocaleUtil.BRAZIL));
+
+		defaultWorkflowTransition.setLabelMap(
+			HashMapBuilder.put(
+				LocaleUtil.US, "Business Rule Review"
```

Comment:
Question:
I notice that, in the labelMap you set the key passing `LocaleUtil.US`
In the [Assertions](https://github.com/liferay-workflow/liferay-portal/pull/1039/commits/a4624e158724a060714b923d17c2edffdba76435#diff-cc610839626999a12eec1cb669c5d88281d74ed45454a67a91ba8a46cee5827dR52-R54) and [mockito](https://github.com/liferay-workflow/liferay-portal/pull/1039/commits/a4624e158724a060714b923d17c2edffdba76435#diff-cc610839626999a12eec1cb669c5d88281d74ed45454a67a91ba8a46cee5827dR38-R39)
you use the key passing `LocaleUtil.getSiteDefault()`.
Its okay letting this two ways to refer the locale "en-US"?

Reply:
Thank you for noticing :smile: 
I changed to use only `LocaleUtil.getSiteDefault()`.